R-package/README.md

```diff
@@ -499,7 +499,6 @@ Or by using the GitHub CLI, using a command similar to this:
 ```shell
 gh workflow run \
     --repo microsoft/LightGBM \
-    --ref ci/fix-rerun-workflow \
```

Comment:
This was a mistake when I included it in #7072, sorry.

`--ref` is the CLI equivalent of the "use workflow from:" dropdown in the GitHub Actions UI (see screenshot in https://github.com/microsoft/LightGBM/pull/7068#issuecomment-3424788828).

It controls which branch to source `r_valgrind.yml` from.

In most cases, that should just be the default branch in the repo. The `--ref ci/fix-rerun-workflow` here was left over from my testing on #7072.